### PR TITLE
Update Readme for more code

### PR DESCRIPTION
**Pull request by MrChengmo**
Update Readme for more code example

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,32 +64,146 @@ import os
...
+        "prompt_tokens": 25,
+        "completion_tokens": 76,
+        "total_tokens": 101
+    }
+}
+```
+
+### 2. 调用能力组件
+- SDK提供了40+个源于百度生态的优质组件，列表可见[组件列表](https://cloud.baidu.com/doc/AppBuilder/s/Glqb6dfiz#3%E3%80%81%E5%BC%80%E9%80%9A%E7%BB%84%E4%BB%B6%E6%9C%8D%E5%8A%A1), 调用前需要申领[免费试用额度](https://console.bce.baidu.com/ai/#/ai/apaas/overview/resource/getFree)
+- 示例中的组件为`百度搜索RAG_PRO组件`, 结合百度搜索的搜索引擎技术和ERNIE模型的语义理解能力，可以更准确地理解用户的搜索意图，并提供与搜索查询相关性更高的搜索结果
```
Comment (guru4elephant) [inline README.md:98]: RAG with Baidu Search增强版

**Review by guru4elephant — COMMENTED** [review]
文案修改下

File: README.md [hunk 2]
```diff
@@ -16,11 +16,21 @@
...
-- 配合 `基础组件` & `流程编排`，积木式搭建个性化的Assistant + FunctionCall应用
-- 提供 `API调用` & `交互式窗口` 两种服务化部署方式，支持快速上云，平滑嵌入到你的产品中
+百度智能云千帆AppBuilder-SDK提供了以下AI应用开发者的必备功能：
+
+- **调用**
+    - 调用大模型，可自由调用您在[百度智能云千帆大模型平台](https://qianfan.cloud.baidu.com/)的模型，开发并调优prompt
+    - 调用能力组件，提供40+个源于百度生态的[优质组件](https://cloud.baidu.com/doc/AppBuilder/s/Glqb6dfiz#3%E3%80%81%E5%BC%80%E9%80%9A%E7%BB%84%E4%BB%B6%E6%9C%8D%E5%8A%A1)，赋能Agent应用
+    - 调用AI原生应用，通过[AppBuilderClient](/docs/basic_module/appbuilder_client.md)可访问并管理在百度智能云千帆AppBuilder[网页端](https://console.bce.baidu.com/ai_apaas/app)发布的AI原生应用，并可注册本地函数联动端云组件
+- **编排**
+    - 编排知识流，通过[KnowledgeBase](/docs/basic_module/knowledgebase.md)管理知识库，进行文档及知识切片的增删改查，配合[网页端](https://console.bce.baidu.com/ai_apaas/app)开发产业级的`RAG`应用
```
[the remark this answers is not in the document]
Comment (MrChengmo) [reply README.md]: 已更新

File: README.md (hunk 1 above)
Comment (MrChengmo) [reply README.md]: 已修改

**Pull request merged**